src/kernel/include/shared/cos_config.h

```diff
@@ -32,6 +32,11 @@
 
 #define COS_MEM_KERN_VA_SZ (1 << 24) /* 16 MB from KERN_START_VA + end of kernel image onward */
 
+#define COS_HW_MMIO_START_VA 0xf0000000 /* Assuming all hardware virtual addresses are beyond this */
+#define COS_PHYMEM_MAX_SZ (COS_HW_MMIO_START_VA-(COS_MEM_KERN_START_VA+COS_MEM_KERN_PA)) /* Maximum addressable physical memory */
+
+#define BOOT_KERN_MEMSCAN_MAX (COS_HW_MMIO_START_VA)
```

Comment:
It isn't clear if this type of stuff to should appear in the memory map or not.

Reply:
This is really used to scan up to a max memory address range and not beyond.
I think it should really be up to what's addressable == ULONG_MAX.